source/options.tsx

```diff
@@ -71,6 +72,25 @@ async function init(): Promise<void> {
 		container.prepend(unchecked.closest('.feature')!);
 	}
 
+	// Highlight new features
+	const newFeatures: Record<string, number> = await cache.get<Record<string, number>>('features-new') ?? {};
```

Comment:
```suggestion
	const newFeatures = await cache.get<Record<string, number>>('features-new') ?? {};
```

Also perhaps we need a better key. `features-new` seems to suggest that it's a list of new features while it's more like `features-first-seen-date`